charts/bookkeeper/values/large.yaml

```diff
@@ -0,0 +1,26 @@
+replicas: 6
+
+resources:
+  requests:
+    cpu: 2000m
+    memory: 8Gi
+  limits:
+    cpu: 4000m
+    memory: 16Gi
+
+storage:
+  ledger:
+    className: standard
+    volumeSize: 250Gi
+  journal:
+    className: standard
+    volumeSize: 250Gi
+  index:
+    className: standard
+    volumeSize: 10Gi
+
+jvmOptions:
+  memoryOpts: ["-Xms2g", "-XX:MaxDirectMemorySize=8g"]
```

Comment:
The rule for memory in the Segment Store is like this: 
- `resources.memory.limit > JVM Xmx + MaxDirectMemorySize` 
- `segmentstore.cache_max_size < MaxDirectMemorySize + 2GB`

In these manifests, I don't see JVM `-Xmx` neither Segment Store `cache_max_size` being defined.

Reply:
Sorry, I was referring to the Segment Store but this is the Bookkeeper manifest. Apologize for the confusion.